.prettierignore

```diff
@@ -0,0 +1,8 @@
+.vscode
+build
+coverage
+node_modules
+npm-debug.log
+yarn.lock
```

Comment:
Nit: Would this be covered by the *.lock below? Not blocking for this PR.

Reply:
Good point. I'll remove that locally so it'll be included a future PR.